test/logs_aggregator/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  loki:
+    container_name: localnet-loki
+    image: grafana/loki:latest
+    ports:
+      - 3100:3100
+    volumes:
+      - ./loki-config.yml:/etc/loki/loki-config.yaml
+  promtail:
+    container_name: localnet-promtail
```

Comment:
shorter !! hehe
the docker compose service name is enough if you ask me for the container name

Reply:
Done! :D